### ISSUE TITLE
Add additional links to DB model and experiment query/mutation

### ISSUE THREAD

**Issue by data-sync-user** [mozilla/experimenter#4370]
We need to store "additional links" titles + urls, add the field to the `experimentBySlug` GQL query so it can be consumed, and add to the `updateExperiment` mutation so it can be updated.

┆Issue is synchronized with this [Jira Task](https://jira.mozilla.com/browse/EXP-802)
┆Issue Number: EXP-802

**Comment by data-sync-user** [comment]
➤ Les Orchard commented:

Bumping this up in points a bit - adding a new list field like this touches a variety of things including DB, admin, GQL queries, and GQL mutations that all adds up.

**Comment by data-sync-user** [comment]
➤ Kate Hudson commented:

Some context around why we need "additional links":

  * We want to encourage folks to follow a standard process in which they are filing a DS jira ticket, use a risk mitgation document, experiment design document, and therefore have a standard set of links.
  * We want to display these links in a structured way that other stakeholders looking at the experiment overview will recognize 
  * We want to be able to run queries to see what percentage of folks have or don't have a particular document type
In the future, we will likely have other document types we'll need to add (or decide not to show some obsolete ones). We may need to display some help text around the document types (e.g. how to file a document)

For this reason, I'd keep the links structured v.s. an open markdown text field – however, unlike the existing comps (https://www.figma.com/file/5SLa07XqQb0ttB1ay1gami/Nimbus-Console-UI?node-id=217143%3A15592) I'd go with specifying the titles instead of leaving them open-ended. 

We decided to go with a separate table with a followup to add enums for title – which will add some complexity but support some of what we've described above
